.gitmodules

```diff
@@ -1,4 +0,0 @@
-[submodule "libs/stories-android"]
-	path = libs/stories-android
-	url = ../../Automattic/stories-android.git
-	shallow = true
```

Comment:
🎉 long time coming! Well done 👏

Reply:
Thanks again @oguzkocer for this indeed ❤️, this will be so much better with all submodules gone 🎉